nixos/modules/services/networking/ddclient.nix

```diff
@@ -102,6 +102,22 @@ in
           Method to determine the IP address to send to the dynamic DNS provider.
         '';
       };
+
+      verbose = mkOption {
+        default = true;
```

Comment:
I'm inclined to agree that verbose logging can be disabled by default.

I am using ddclient and had a look at the logs though, from what I can see the verbose logs are not actually particularly verbose, basically my journal contains mostly lines like this
```
Mar 06 15:35:15 beta ddclient[1265]: SUCCESS:  XYZ: skipped: IP address was already set to a.b.c.d.
Mar 06 15:45:15 beta ddclient[1265]: SUCCESS:  XYZ: skipped: IP address was already set to a.b.c.d.
```
I.e., one entry every 10 minutes, I assume disabling verbose logging would remove these lines but I don't exactly worry about the volumes :-)